pkg/v1/tkg/managementcomponents/management_component_install.go

```diff
@@ -196,3 +198,30 @@ func WaitForManagementPackages(clusterClient clusterclient.Client, packageInstal
 	}
 	return nil
 }
+
+// DeletePackageInstall deletes the PackageInstall CR
+func noopDeletePackageInstall(ctx context.Context, clusterClient clusterclient.Client, pkgClient tkgpackageclient.TKGPackageClient, pkgiName, pkgiNamespace string) error {
+	pkgi := &kappipkg.PackageInstall{}
+	err := clusterClient.GetResource(pkgi, pkgiName, pkgiNamespace, nil, nil)
```

Comment:
Need to handle "not found" which is not an error. if not found nothing to do so, return nil